ui/src/components/layout/stepper/stepper.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button } from '../../core';

```

Comment:
Is this file needed even if we don't want to do any storybook work for now?

Reply:
It's not mandatory, but it helps me to build just the component cause first I just did the stepper part, without all the related card and the tiltes and button, so it's helpful for the component building (specially the styling part)